README.md

```diff
@@ -136,6 +136,41 @@ re-executed in order to pick up a non-hermetic change to your environment (e.g.,
 updating your system `python` interpreter), you can completely flush out your
 repo cache with `bazel clean --expunge`.
 
+### Importing `pip` dependencies incrementally (experimental)
```

Comment:
"incrementally" doesn't feel like the right word to me, I would have expected sth like "partial download"

Reply:
I've always felt the same and said the same in the original issue: https://github.com/bazelbuild/rules_python/issues/395

`Fetching pip dependencies lazily (experimental)`?

Lazy vs Eager?
Ahead of time vs Just in time?

Reply:
I was going off the convention in that issue, but I agree incrementally doesn't feel very explanatory. @alexeagle suggested something more like "parse_pip_{deps|requirements}" (into starlark functions) which makes more sense with the follow up install_deps() call.